src/main/java/ingredientes/Fruta.java

```diff
@@ -15,7 +15,7 @@ public TipoFruta getTipoFruta(){
      //É necessário consertar o compareTo, para imprimir na ordem correta.
      @Override
      public int compareTo(Ingrediente ingrediente) {
-          return ingrediente.obterTipo().toString().compareTo(this.obterTipo().toString());
+          return ingrediente.obterTipo().toString().compareTo(this.obterTipo().toString()) * -1;
```

Comment:
o return ficou invertido, era necessário arrumar:

return **this**.obterTipo().toString().compareTo(**ingrediente**.obterTipo().toString());